luigi/contrib/gcs.py

```diff
@@ -249,6 +249,28 @@ def put(self, filename, dest_path, mimetype=None, chunksize=None):
 
         self._do_put(media, dest_path)
 
+    def _put_star(self, kwargs):
```

Comment:
I'm not sure this hidden method name makes intuitive sense. Why not just call it `_put(self, kwargs)`?

Reply:
The only purpose is to expand the kwargs because the multiprocessing `map` method does not allow to pass multiple parameters. 

I would think the `_put(self, kwargs)` method is the hidden implementation of the `put` method, which it's not. A more explicit name name could be `_forward_args_to_put` maybe?